tests/server-common/src/test/java/io/a2a/server/apps/common/AbstractA2AServerTest.java

```diff
@@ -370,517 +241,41 @@ public void testError() {
                 .taskId(SEND_MESSAGE_NOT_SUPPORTED.getId())
                 .contextId(SEND_MESSAGE_NOT_SUPPORTED.getContextId())
                 .build();
-        SendMessageRequest request = new SendMessageRequest(
-                "1", new MessageSendParams(message, null, null));
-        SendMessageResponse response = given()
-                .contentType(MediaType.APPLICATION_JSON)
-                .body(request)
-                .when()
-                .post("/")
-                .then()
-                .statusCode(200)
-                .extract()
-                .as(SendMessageResponse.class);
-        assertEquals(request.getId(), response.getId());
-        assertNull(response.getResult());
-        // this should be an instance of UnsupportedOperationError, see https://github.com/a2aproject/a2a-java/issues/23
-        assertInstanceOf(JSONRPCError.class, response.getError());
-        assertEquals(new UnsupportedOperationError().getCode(), response.getError().getCode());
-    }
-
-    @Test
-    public void testGetAgentCard() {
-        AgentCard agentCard = given()
-                .contentType(MediaType.APPLICATION_JSON)
-                .when()
-                .get("/.well-known/agent.json")
-                .then()
-                .statusCode(200)
-                .extract()
-                .as(AgentCard.class);
-        assertNotNull(agentCard);
-        assertEquals("test-card", agentCard.name());
-        assertEquals("A test agent card", agentCard.description());
-        assertEquals("http://localhost:8081", agentCard.url());
-        assertEquals("1.0", agentCard.version());
-        assertEquals("http://example.com/docs", agentCard.documentationUrl());
-        assertTrue(agentCard.capabilities().pushNotifications());
-        assertTrue(agentCard.capabilities().streaming());
-        assertTrue(agentCard.capabilities().stateTransitionHistory());
-        assertTrue(agentCard.skills().isEmpty());
-    }
-
-    @Test
-    public void testGetExtendAgentCardNotSupported() {
-        given()
-                .contentType(MediaType.APPLICATION_JSON)
-                .when()
-                .get("/agent/authenticatedExtendedCard")
-                .then()
-                .statusCode(404)
-                .body("error", equalTo("Extended agent card not supported or not enabled."));
-    }
-
-    @Test
-    public void testMalformedJSONRPCRequest() {
```

Comment:
This test should be added back. Since this is testing a malformed request, you can leave the test as it was before, no need to use A2A client for this one.